src/components/Composer/implementation/index.native.tsx

```diff
@@ -140,7 +140,11 @@ function Composer(
             textAlignVertical="center"
             style={[composerStyle, maxHeightStyle]}
             markdownStyle={markdownStyle}
-            autoFocus={autoFocus}
+            // /*
+            // There are cases in hybird app on android that screen goes up when there is autofocus on keyboard. (e.g. https://github.com/Expensify/App/issues/53185)
+            // Workaround for this issue is to maunally focus keyboard after it's acutally rendered which is done by useAutoFocusInput hook.
+            // */
+            autoFocus={Platform.OS !== 'android' ? autoFocus : false}
```

Comment:
@jnowakow Because we already used useAutoFocusInput, could we remove this line?
```suggestion
```

Reply:
Without this line iOS doesn't work. It's quite strange issue 

https://github.com/user-attachments/assets/86b76cb6-d8a7-40b0-bfb7-8564c819ca11

Reply:
@jnowakow Thanks. It doesn't work on IOS only. I think it relates to this problem: https://github.com/facebook/react-native/issues/47576

Reply:
Sounds like it may be this problem